harpy/metassembly.py

```
@@ -0,0 +1,104 @@
+    if hpc:
+        command += f"--workflow-profile {hpc} "
+    if snakemake:
+        command += snakemake
+
+    validate_fastq_bx([fastq_r1, fastq_r2], threads, quiet)
+    os.makedirs(workflowdir, exist_ok=True)
+    fetch_rule(workflowdir, f"metassembly.smk")
+    os.makedirs(f"{output_dir}/logs/snakemake", exist_ok = True)
+    sm_log = snakemake_log(output_dir, "metassembly")
+    configs = {
+        "workflow" : asm,
```

Ruff

Undefined name `asm`

(F821)